feeds/wlan-ap/wlan-ap-config/files/etc/uci-defaults/99-tip-data

```diff
@@ -2,22 +2,85 @@
 
 . /lib/functions.sh
 
+SKU="unknown"
+MODEL="unknown"
+PLATFORM="unknown"
+SERIAL="unknown"
+MODEL_REV="unknown"
+MODEL_DESCR="unknown"
+MANUF_NAME="unknown"
+MANUF_DATE="unknown"
+MANUF_URL="unknown"
+MANUF_DESIGN="unknown"
+REF_DESIGN="unknown"
+CERT_REGION="unknown"
+ID=""
+
 case "$(board_name)" in
 edgecore,ecw5211|\
 edgecore,ecw5410)
 	MODEL=$(cat /tmp/sysinfo/board_name | sed "s/edgecore,//" | tr [a-z] [A-Z])
-	SERIAL=$(cat /sys/class/net/eth0/address | tr -d :)
 	PLATFORM=$(cat /tmp/sysinfo/model)
+	SERIAL=$(cat /dev/mtd5 | grep serial_number | cut -d "=" -f2)
+	SKU=$(cat /dev/mtd5 | grep sku | cut -d "=" -f2)
+	CERT_REGION=$(cat /dev/mtd5 | grep certification_region | cut -d "=" -f2)
+	ID=$(cat /dev/mtd5 | grep mac_address | cut -d "=" -f2)
+	MANUF_DATE=$(cat /dev/mtd5 | grep manufacturer_date | cut -d "=" -f2)
+	MANUF_NAME="Edge-Core"
```

Comment:
Based on the packaging and corporate website, they don't capitalize the "c" in "core" and they do capitalize the "E.  So "Edge-corE".  Discussed with Marcel and he agrees.

Reply:
I see below from the flash of one of the ECW5211 I received recently. As we are hard-coding the name string. We can go ahead with "Edge-corE". The first and last letters are capitalized. Correct? Let me know.

root@OpenAp-28e498:~# cat /dev/mtd5 | grep manuf
manufacturer_date=11-13-2020
manufacturer_name=Edge-Core
manufacturer_url=www.edge-core.com

Reply:
I see in the web-page that they are supposed to provide this value and that the expected name is listed.  Therefore, I would not hard-code these values.  I would read the value provided, and use the default in the table if it is not found.

Reply:
Makes sense. I will that change for all the platforms then.
- Will read the "manufacturer_name" from flash and commit to UCI.
- If not found in flash, hard-code it to the string from the table in https://telecominfraproject.atlassian.net/wiki/spaces/WIFI/pages/353337353/AP+Manufacturing+Data

Should we update the table in the above spec for Edge-Core to Edge-corE ??

Reply:
Updated the changes to read and present the flash content for "manufacturer name", and use the table default if not present.

Reply:
Leave Edge-Core as-is; they have had time to review the Confluence page and change if they wanted.